pages/posts/analysis-of-json-schema-defects.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "An Analysis of JSON Schema Defects"
+date: 2023-09-10
+tags:
+  - Specification
+type: Opinion
+cover: /img/posts/2023/analysis-of-json-schema-defects/cover.webp
+authors:
+  - name: Fabien Coelho
+    photo: /img/avatars/fabien.jpg
+    link: https://www.linkedin.com/in/fabien-coelho-65433a18/
+    byline: Professor in Computer Science
+  - name: Claire Medrala
+    photo: /img/avatars/claire.jpg
+    link: https://www.linkedin.com/in/claire-medrala/
+    byline: Research Engineer
+excerpt: Evidences show that schemas are hard to write, and suggest changes in the spec
+---
+
+We have studied the quality of available public schemas
+(see [paper](https://www.cri.minesparis.psl.eu/classement/doc/A-794.pdf)) through static analysis.
+Over **60%** of schemas are shown to have some type of defects, resulting in
+unintended data to be validated, risking system breakage or even cybersecurity issues.
+These findings suggest key changes in JSON Schema specification which would block most
+of encountered defects.
```

Comment:
I disagree with the conclusion that because people aren't using JSON Schema correctly (in many cases, they're typographical errors) that JSON Schema is at fault and needs to change.